Locally i still run into errors, when running make test. 

```bash
package command-line-arguments
        imports machine
        imports device/stm32: build constraints exclude all Go files in /home/tobias/.cache/tinygo/goroot-go1.16-65176e9981f9146086a1cca207bd1d47f4da8a15455d548a5d84fac12d1814d2-syscall/src/device/stm32
make: *** [Makefile:36: smoke-test] Fehler 1
```

--- a/Makefile
+++ b/Makefile
@@ -166,7 +166,7 @@ endif
 	tinygo build -size short -o ./build/test.hex -target=circuitplay-express ./examples/lis2mdl/main.go
 	@md5sum ./build/test.hex
 	tinygo build -size short -o ./build/test.hex -target=feather-m0 ./examples/dht/main.go
-	@md5sum ./build/t
+	@md5sum ./build/test.hex
 	tinygo build -size short -o ./build/test.hex -target=arduino ./examples/keypad4x4/main.go
 	@md5sum ./build/test.hex
 

